packages/react-instantsearch/examples/react-router-v4/src/App.js

```diff
@@ -7,33 +7,35 @@ import {
 import 'react-instantsearch-theme-algolia/style.css';
 import qs from 'qs';
 
-export default class App extends Component {
-  constructor (props) {
+const updateAfter = 700;
+const searchStateToUrl =
+  (props, searchState) =>
+    searchState ? `${props.location.pathname}?${qs.stringify(searchState)}` : '';
```

Comment:
I extracted this method:

```js
const createURL = (state) => `?${qs.stringify(state)}`
```

for be used lately and avoid repeat this pice of code 👍